tests/ui/undocumented_unsafe_blocks.rs

```diff
@@ -89,11 +93,6 @@ fn block_comment_newlines() {
     unsafe {}
 }
 
-#[rustfmt::skip]
-fn inline_block_comment() {
-    /* Safety: */unsafe {}
```

Comment:
It would be better to leave this as it is and add comments not to support this style.